common/environment/environment.cpp

```diff
@@ -1630,7 +1626,9 @@ void CLocalEnvironment::clearCache()
 IConstDropZoneInfo * CLocalEnvironment::getDropZoneByAddressPath(const char * netaddress, const char *targetFilePath) const
 {
     IConstDropZoneInfo * dropZone = nullptr;
+    IConstDropZoneInfo * candidateDropZone = nullptr;
```

Comment:
trivial: slightly clearer if declared inside strncmp block, as set at top and not used outside that block.